src/app/hooks/trading/useSwap_RecentTrades.ts

```diff
@@ -1,36 +1,32 @@
-import { useState, useEffect } from 'react';
-import axios from 'axios';
-import { backendUrl, currentChainId } from 'utils/classifiers';
-import { RecentSwapsDataEntry } from 'types/trading-pairs';
+import { useEffect, useMemo } from 'react';
 import { Asset } from 'types';
-import { getTokenContract } from 'utils/blockchain/contract-helpers';
-
-const url = `${backendUrl[currentChainId]}/recentEvents/swap`;
+import { RecentSwapsDataEntry } from 'types/trading-pairs';
+import { APOLLO_POLL_INTERVAL } from 'utils/classifiers';
+import { LiquidityPoolDictionary } from 'utils/dictionaries/liquidity-pool-dictionary';
+import { useGetRecentSwapEventsLazyQuery } from 'utils/graphql/rsk/generated';
 
 export const useSwap_RecentTrades = (baseToken: Asset, quoteToken: Asset) => {
-  const [data, setData] = useState<RecentSwapsDataEntry[] | undefined>(
-    undefined,
+  const pool = useMemo(
+    () => LiquidityPoolDictionary.get(baseToken, quoteToken),
+    [baseToken, quoteToken],
   );
-  const baseTokenAddress = getTokenContract(baseToken).address;
-  const quoteTokenAddress = getTokenContract(quoteToken).address;
-  const [loading, setLoading] = useState(false);
+
+  const [load, { called, data, loading }] = useGetRecentSwapEventsLazyQuery({
+    variables: {
+      converterAddress: [pool.converter].map(item => item.toLowerCase()),
```

Comment:
```suggestion
      converterAddress: [pool.converter.toLowerCase()],
```
Why you did it via `map` before?

Reply:
Ops, there was multiple addresses before :)